LICENCE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Please rename this file back to LICENSE (with an S) as everywhere in this license, and in the license part of the readme it's consistently being spelled with an S and not with two C's.